src/variants/GHOST_ATTO/variant.h

```diff
@@ -27,29 +27,29 @@ extern "C" {
  *        Pins
  *----------------------------------------------------------------------------*/
 
-#define PA10 0
-#define PA9  1
-#define PA12 2
-#define PB0  3
-#define PB7  4
-#define PB6  5
-#define PB1  6
-#define PF0  7 // XO, don't use!
-#define PF1  8 // XO, don't use!
-#define PA8  9
-#define PA11 10
-#define PB5  11
-#define PB4  12
-#define PB3  13
-#define PA0  14
-#define PA1  15
-#define PA3  16
-#define PA4  17
-#define PA5  18 // if SB18 ON (default) connected to PB7
-#define PA6  19 // if SB16 ON (default) connected to PB6
-#define PA7  20
-#define PA2  21 // STLink Tx
-#define PA15 22 // STLink Rx
+#define PA10 0    //
```

Comment:
These comments does not belongs to here. These variant names (mcu type specific) should be changed to more generic in the future to avoid copy code.